src/components/super-spawner.js

```diff
@@ -1,112 +1,127 @@
+import { addMedia } from "../utils/media-utils";
+import { waitForEvent } from "../utils/async-utils";
+
+let nextGrabId = 0;
 /**
  * Spawns networked objects when grabbed.
  * @namespace network
  * @component super-spawner
  */
 AFRAME.registerComponent("super-spawner", {
   schema: {
-    template: { default: "" },
+    /**
+     * Source of the media asset the spawner will spawn when grabbed. This can be a gltf, video, or image, or a url that the reticiulm media API can resolve to a gltf, video, or image.
+     */
+    src: { default: "https://asset-bundles-prod.reticulum.io/interactables/Ducky/DuckyMesh-438ff8e022.gltf" },
```

Comment:
having this defined here as the default seems wrong- this should get moved into the environment component data

Reply:
Yep, the plan would be to update the enviornments after we push this, then we can remove this default. This is just to allow us to push without breaking the existing environments during out push.